### PR TITLE
Pipe bomb

### PR DESCRIPTION
**Pull request by TheChosenEvilOne**
## About The Pull Request
Adds capped pipes and pipe bombs. Capped pipes are a 40 volume beaker you can weld to seal and if you add a fuse (cable) it will become a pipe bomb. Pipe bomb when opened from a package has an 80% chance of detonating, and if the pipe bomb explodes it will create shrapnel.

## Why It's Good For The Game
1. A way to make ghetto grenades.
2. You can send a mail bomb to the HoS.
3. it is funny.

## Changelog
:cl:
add: Added capped pipes, a 40 volume beaker made from welding a straight pipe. If welded again it will be capped and can't be spilled/filled/emptied. If you add a fuse (cable coil) before sealing the pipe it will become a pipe bomb, when detonated it heats up reagent contents by a random amount from five to twenty degrees.
/:cl:

### CODE REVIEW

**Comment by TheChosenEvilOne** [comment]
oh yeah, this needs a proper sprite for the fuse because right now when you activate the bomb it just says "fire" on it.

**Comment by TheChosenEvilOne** [comment]
this is now ready.

File: code/game/objects/items/pipe_bomb.dm [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	icon_state = "pipe"
+	volume = 40
+	reagent_flags = REFILLABLE | DRAINABLE | AMOUNT_VISIBLE
+	spillable = TRUE
+	w_class = WEIGHT_CLASS_NORMAL
+	var/shrapnel_type = /obj/projectile/bullet/shrapnel
+	var/shrapnel_magnitude = 2
+	var/sealed = FALSE
+	var/fused = FALSE
+	var/fuse_length = 0
```
Comment (Canohaaaaaaa) [inline code/game/objects/items/pipe_bomb.dm:26]: Doc fuse length

Comment (TheChosenEvilOne) [reply code/game/objects/items/pipe_bomb.dm:28]: done

**Pull request merged**